src/collections/unordered-map.ts

```diff
@@ -40,19 +40,19 @@ export class UnorderedMap {
     return keysIsEmpty;
   }
 
-  serializeIndex(index: number): Bytes {
+  private serializeIndex(index: number): Bytes {
     let data = new Uint32Array([index]);
     let array = new Uint8Array(data.buffer);
     return u8ArrayToBytes(array);
   }
 
-  deserializeIndex(rawIndex: Bytes) {
+  private deserializeIndex(rawIndex: Bytes) {
     let array = bytesToU8Array(rawIndex);
     let data = new Uint32Array(array.buffer);
     return data[0];
   }
 
-  getIndexRaw(key: Bytes): Bytes {
+  private getIndexRaw(key: Bytes): Bytes {
     let indexLookup = this.keyIndexPrefix + JSON.stringify(key);
     let indexRaw = near.storageRead(indexLookup);
     return indexRaw;
```

Comment:
why are these associated functions to begin with? Can't these live outside of the structure to avoid misuse (when ignoring or not using TS types)?